lib/matplotlib/offsetbox.py

```diff
@@ -1722,7 +1718,18 @@ def __init__(self, ref_artist, use_blit=False):
         c2 = self.canvas.mpl_connect('pick_event', self.on_pick)
         c3 = self.canvas.mpl_connect('button_release_event', self.on_release)
 
-        ref_artist.set_picker(self.artist_picker)
+        if not ref_artist.pickable():
+            ref_artist.set_picker(True)
```

Comment:
Additionally, I left the previous picker in place if the user already configured their own.